lstchain/mc/tests/test_senstivity.py

```diff
@@ -46,25 +46,25 @@ def test_calculate_sensitivity_lima():
     
     np.testing.assert_allclose(calculate_sensitivity_lima(
             50, 10, 0.2, 1, 0, 0),
-                               ([13.48, 26.97]), rtol = 1.e-3)
+                               ([13.49, 26.97]), rtol = 1.e-3)
     np.testing.assert_allclose(calculate_sensitivity_lima(
             200, 50, 1, 0, 1, 0),
-                               ([63.00, 31.5]), rtol = 1.e-3)
+                               ([63.01, 31.50]), rtol = 1.e-3)
     # Testing an array
     np.testing.assert_allclose(calculate_sensitivity_lima(
```

Comment:
Why were these values changed? Shouldn't they work like before now?

Reply:
they were essentially the same within the `rtol`, but anyways I returned them to the old values now

Reply:
No, i mean the values below, which changed not just at some decimal digits

Reply:
I changed some of the input values of the tests because there were some `nan`, I'm guessing because of low number of `on/off` events